src/test/kotlin/uk/gov/justice/digital/hmpps/managerecallsapi/component/CreateDossierComponentTest.kt

```diff
@@ -55,7 +47,8 @@ class CreateDossierComponentTest : ComponentTestBase() {
 
     val dossier = authenticatedClient.getDossier(recall.recallId)
 
-    assertThat(dossier.content, equalTo(expectedBase64NumberedPdf))
+//    File("test-results.pdf").writeBytes(Base64.getDecoder().decode(dossier.content))
+    assertThat(dossier, hasNumberOfPages(equalTo(3)))
```

Comment:
Trying to work out what this assert is really testing: asserting on `hasNumberOfPages` looks a bit misleading here?
And if we keep it I wonder if using this is misleading? 
`expectedMergedPdf = ClassPathResource("/document/recall-notification.pdf")`

Reply:
Yeah we don't really assert on much here because we can't as we stub the response from gotenberg.  Maybe just a successful response is sufficient